x-pack/elastic-agent/pkg/agent/operation/operation_install.go

```diff
@@ -8,7 +8,8 @@ import (
 	"context"
 	"os"
 
-	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/agent/errors"
+	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/core/plugin/state"
+
```

Comment:
same everywhere

Reply:
I ran `mage fmt`, it passed lint as well, so it believe this is valid. But I will fix.